test/JoeMakerV4.test.ts

```diff
@@ -125,7 +125,7 @@ describe("joeMakerV4", function () {
       await this.router.swapExactAVAXForTokens("0", [this.wavax.address, this.usdc.address], this.joeMakerV4.address, DEADLINE, {
         value: ethers.utils.parseEther("2"),
       })
-      await this.joeMakerV4.convert(this.usdc.address, this.usdc.address)
+      await this.joeMakerV4.convert(this.usdc.address, this.usdc.address, "100")
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.wavaxERC20.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdc.balanceOf(BAR_ADDRESS)).to.equal("215434297")
```

Comment:
Don't get what you're testing here. Shouldn't it test that USDC was converted to JOE and the balance of JOE held by Bar was increased by an expected amount?